docs/modelserving/v1beta1/custom/custom_model/README.md

```diff
@@ -43,7 +43,7 @@ to run the image build on the cloud and continuously build/deploy new versions f
 
 ### Use pack to build and push the custom model server image
 ```bash
-pack build --builder=heroku/buildpacks:20 ${DOCKER_USER}/custom-model:v1
+pack build --builder paketobuildpacks/builder:base ${DOCKER_USER}/custom-model:v1
```

Comment:
@samj1912 getting the following error with paketobuildpack, it is working fine with heroku buildpack or manually crafted Dockerfile.
```
Traceback (most recent call last):
  File "/layers/paketo-buildpacks_cpython/cpython/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/layers/paketo-buildpacks_cpython/cpython/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/workspace/model.py", line 15, in <module>
    import kserve
  File "/layers/paketo-buildpacks_pip-install/packages/lib/python3.8/site-packages/kserve/__init__.py", line 17, in <module>
    from kserve.model_server import ModelServer
  File "/layers/paketo-buildpacks_pip-install/packages/lib/python3.8/site-packages/kserve/model_server.py", line 27, in <module>
    import kserve.handlers as handlers
  File "/layers/paketo-buildpacks_pip-install/packages/lib/python3.8/site-packages/kserve/handlers/__init__.py", line 15, in <module>
    from .base import NotFoundHandler  # noqa # pylint: disable=unused-import
  File "/layers/paketo-buildpacks_pip-install/packages/lib/python3.8/site-packages/kserve/handlers/base.py", line 18, in <module>
    from kserve.model_repository import ModelRepository
  File "/layers/paketo-buildpacks_pip-install/packages/lib/python3.8/site-packages/kserve/model_repository.py", line 17, in <module>
    from ray.serve.api import RayServeHandle
  File "/layers/paketo-buildpacks_pip-install/packages/lib/python3.8/site-packages/ray/serve/__init__.py", line 2, in <module>
    from ray.serve.api import (start, get_replica_context, shutdown, ingress,
  File "/layers/paketo-buildpacks_pip-install/packages/lib/python3.8/site-packages/ray/serve/api.py", line 25, in <module>
    from ray.serve.controller import ServeController
  File "/layers/paketo-buildpacks_pip-install/packages/lib/python3.8/site-packages/ray/serve/controller.py", line 13, in <module>
    from ray.serve.deployment_state import ReplicaState, DeploymentStateManager
  File "/layers/paketo-buildpacks_pip-install/packages/lib/python3.8/site-packages/ray/serve/deployment_state.py", line 19, in <module>
    from ray.serve.storage.kv_store import KVStoreBase
  File "/layers/paketo-buildpacks_pip-install/packages/lib/python3.8/site-packages/ray/serve/storage/kv_store.py", line 1, in <module>
    import sqlite3
  File "/layers/paketo-buildpacks_cpython/cpython/lib/python3.8/sqlite3/__init__.py", line 23, in <module>
    from sqlite3.dbapi2 import *
  File "/layers/paketo-buildpacks_cpython/cpython/lib/python3.8/sqlite3/dbapi2.py", line 27, in <module>
    from _sqlite3 import *
ImportError: libsqlite3.so.0: cannot open shared object file: No such file or directory
```

Reply:
@yuzisun it should work now.